app/backend/prepdocslib/pdfparser.py

```diff
@@ -107,5 +198,28 @@ def table_to_html(cls, table: DocumentTable):
                     cell_spans += f" rowSpan={cell.row_span}"
                 table_html += f"<{tag}{cell_spans}>{html.escape(cell.content)}</{tag}>"
             table_html += "</tr>"
-        table_html += "</table>"
+        table_html += "</table></figure>"
         return table_html
+
+    @staticmethod
+    def crop_image_from_pdf_page(doc: pymupdf.Document, page_number, bounding_box) -> bytes:
+        """
+        Crops a region from a given page in a PDF and returns it as an image.
+
+        :param pdf_path: Path to the PDF file.
+        :param page_number: The page number to crop from (0-indexed).
+        :param bounding_box: A tuple of (x0, y0, x1, y1) coordinates for the bounding box.
+        :return: A PIL Image of the cropped area.
+        """
+        page = doc.load_page(page_number)
+
+        # Cropping the page. The rect requires the coordinates in the format (x0, y0, x1, y1).
+        bbx = [x * 72 for x in bounding_box]
+        rect = pymupdf.Rect(bbx)
+        # 72 is the DPI ? what? explain this from CU
```

Comment:
let's fill this in:

Bound box is scaled to 72 dots per inch
Assume the PDF has 300 DPI
The matrix is used to convert between these 2 units